### DISCUSSION TITLE
Oracle Cloud port or address fail

### DISCUSSION THREAD

**Discussion by Crissomar** [itzg/docker-minecraft-bedrock-server#425] [Q&A]
So I have been playing with Oracle Cloud free tier, more specifically with Oracle Linux on arm
I got one java server running **natively** and I can connect freely
But when I try to run this docker image with the following yml simply does not work
```Dockerfile
services:
  bds:
    image: itzg/minecraft-bedrock-server
    environment:
      GAMEMODE: survival
      EULA: "TRUE"
      SERVER_NAME: ServerDeMine
      DIFFICULTY: normal
      ONLINE_MODE: false
    ports:
      - "19132:19132/udp"
      - "19132:19132/tcp"
    volumes:
      - ./data:/data
    stdin_open: true
    tty: true
```
in the oracle panel I have open ports for java and bedrock
I think its a problem with my docker or container configuration, I do not know really...

One idea could be that the forwarded address and the linked address on the container are different but idk so much about the BDS or Docker

**Comment by itzg** [comment]
There is nothing here I can fix. Your container looks right other than the unnecessary TCP port mapping.

(I can convert this to a discussion if you want it to stay open for others to help answer.)

**Comment by Crissomar** [comment]
please do 🙏
